src/frontend/sharedComponents/HomeHeader.tsx

```diff
@@ -2,21 +2,27 @@ import React from 'react';
 import {View, StyleSheet} from 'react-native';
 import LinearGradient from 'react-native-linear-gradient';
 import {IconButton} from './IconButton';
-import {ObservationListIcon} from './icons';
+import {ObservationListIcon, SyncIconCircle} from './icons';
 import {useNavigationFromHomeTabs} from '../hooks/useNavigationWithTypes';
-import {Text} from './Text';
+import {useWifiName} from '../hooks/useWifiStatus';
 
 export const HomeHeader = () => {
   const navigation = useNavigationFromHomeTabs();
+  const ssid = useWifiName();
 
   return (
     <View style={[styles.header]}>
       <LinearGradient
         style={styles.linearGradient}
         colors={['#0006', '#0000']}
       />
-      {/* Stand in for styling,  */}
-      <Text>""</Text>
+      <IconButton
+        style={styles.leftButton}
+        onPress={() => {
+          navigation.navigate(ssid ? 'Sync' : 'NoWifi');
```

Comment:
yeah really not sure about using screens for wifi state - it's going to confuse the navigation stack a lot, and it means that lots of screens need to know about wifi state so it breaks encapsulation (adding a sync button to anywhere requires adding ssid state to that component where you add it)

Reply:
I was just trying to leverage react navigation animations for transition between screens. Without it, it is pretty harsh, but I think having no animation is ok for the MVP, and then we can create a custom animation later